src/main/java/com/redhat/documentation/asciidoc/Util.java

```diff
@@ -40,4 +40,9 @@ public static String fixAsset(String line) {
         return line
                 .replaceAll("(video|audio)::(\\w+)\\.(\\w+)\\[(.*)]", "$1::" + ASSETS_LOCATION + "/$2.$3[$4]");
     }
+
+    public static String fixinsideAsset(String line) {
+        return line
+                .replaceAll("(video|audio)::(\\w+)\\/(\\w+)\\.(\\w+)\\[(.*)]", "$1::" + ASSETS_LOCATION + "/$2/$3.$4[$5]");
```

Comment:
I don't see how this does anything different than `fixAsset`. If something does need to get fixed in how `fixAsset` works to handle the directory structure, it would be better to do it there.